lib/signex/helper.ex

```diff
@@ -11,6 +11,17 @@ defmodule SignEx.Helper do
     :public_key.pem_entry_decode(key_entry)
   end
 
+  def encryption_type(decoded_key) do
+    case decoded_key do
+      {:RSAPublicKey, _, _} ->
+        :rsa
+      {{:ECPoint, _}, {:namedCurve, _curve_tuple}} ->
+        # we could potentially use the curve tuple to make sure
```

Comment:
I think this comment should be a TODO or removed ?

Reply:
I don't think it's really a TODO, just my notes from digging into the `:public_key` api. If we ever do start getting more specific on the curves we accept, it could be helpful to know that it's here.

Reply:
My vote would be to match on the curl and relax if need be in the future, as long as we can expose a useful message about why our strict code rejected something